### PR TITLE
Added CowDeathTable react component, storybook, fixures and tests

### LINKED ISSUE #50: Add cow death table
Component 3 of https://github.com/ucsb-cs156-f22/f22-happycows-issues/issues/17

# Acceptance Criteria
- [ ] Add a new React component `CowDeathTable`
  - [ ] `CowDeathTable` displays a list of `CowDeath`s
  - `CowDeath` has fields:
    - `id`
    - `commonsId`
    - `userId`
    - `ZonedDateTime` (typo? TODO: inquire with team 4 about cow death timestamp field name)
    - `cowsKilled`
    - `avgHealth`
- [ ] Add `cowDeathTableFixtures`
- [ ] Add tests for `CowDeathTable`
- [ ] Add storybook entry for `CowDeathTable`

### PR DESCRIPTION
**Pull request by minhbui27**
In this commit, I added the CowDeathTable react component (frontend) only with no endpoints (those are for other issues), a storybook component along with fixtures, and tests.

Closes #50 

Storybook entry: 
https://ucsb-cs156-f22.github.io/f22-7pm-happycows-docs-qa/storybook-qa/Minh-CowDeathTable/?path=/story/components-cowdeaths-cowdeathtable--five-user-commons

Below are images as shown via the storybook
image
image
image
image

### CODE REVIEW

**Review by RAILGUN1124 — APPROVED** [review]
I approve

File: frontend/src/fixtures/CowDeathFixtures.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+const cowDeathFixtures = {
+    oneUserCommonsCD: 
+    [
+        {
+            "id":1,
+            "commonsId": 1,
+            "userId": 1,
+            "createdAt" : "2022-11-13T19:50:16.810569-08:00",
+            "cowsKilled": 3,
+						"avgHealth": 20
```
Comment (ethwu) [inline frontend/src/fixtures/CowDeathFixtures.js:10]: Probably should replace these tabs with 4 spaces

**Review by blakerandle — CHANGES_REQUESTED** [review]
I am confused by why the header for one of the columns in CowDeathTable.js is 'ZonedDateTime' rather than something like 'Created At.' I understand it is technically in the spec but it seems like a typo and would be confusing to a user.

**Review by blakerandle — APPROVED** [review]
lgtm

**Review by ethwu — APPROVED** [review]

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**